test/integration/auth/svcaccttoken_test.go

```diff
@@ -306,6 +306,8 @@ func TestServiceAccountTokenCreate(t *testing.T) {
 		checkPayload(t, treq.Status.Token, "null", "kubernetes.io", "node")
 
 		info := doTokenReview(t, cs, treq, false)
+		// we are not testing the credential-id feature, so delete this value from the returned extra info map
+		delete(info.Extra, apiserverserviceaccount.CredentialIDKey)
```

Comment:
Why not simply disable that feature gate during this test?

Reply:
Once the feature goes GA it won't be possible to disable it, so tried to solve for the future a little bit